nowcasting_dataset/data_sources/pv/pv_data_source.py

```diff
@@ -47,14 +47,20 @@ class PVDataSource(ImageDataSource):
 
     def __post_init__(self, image_size_pixels: int, meters_per_pixel: int):
         """Post Init"""
+        if not isinstance(self.filenames, list):
```

Comment:
In general, I'd recommend that we allow `filenames` to be _either_ a `list` or a `tuple`.

```suggestion
        if not isinstance(self.filenames, (tuple, list)):
```

Also, to keep the code as simple as possible, I'd maybe recommend that `PVDataSource` simply blows up if `self.filenames` isn't a `tuple` or a `list`!  (Are there any use-cases where `self.filenames` might still be a `str`?)